plugin.js

```diff
@@ -159,6 +168,12 @@ function casPlugin (server, options) {
       username: session.username,
       attributes: session.attributes
     }
+    if (_options.value.sessionCredentialsMappings) {
+      const dotProp = require('dot-prop')
```

Comment:
Imports lead the file (stdlib first, third party second, and local third) unless they need to be scoped to a function. This one doesn't.